EasyInventory/src/main/java/com/easyinventory/enterprise/service/EasyInventoryServiceIMPL.java

```diff
@@ -30,7 +34,14 @@ public EasyInventoryServiceIMPL(CategoryRepository categoryRepository, ItemRepos
      */
     @Override
     public Category saveCategory(Category category) throws Exception {
-        return categoryRepository.save(category);
+        try {
```

Comment:
Throughout this entire class, I added try-catch blocks to the methods that were throwing exceptions. Additionally, I logged info when needed and an error when an exception was caught.